### ISSUE TITLE
add previous user agents

### ISSUE THREAD

**Issue by howardmrhodes** [misaka00251/random-agent-spoofer#2]
Can you please add all of the older useragents from previous versions of RAS (0.9.8.5.3 onwards) in the next release as it gives many more options to choose from
or can you let me know how to do that myself (pointing to a link or tutorial would be enough)

**Comment by misaka00251** [comment]
Sorry, will do that in the next release, as I didn't check the notification.

You need to know how to write json, and know basic stuff about UA.

**Comment by misaka00251** [comment]
Well, I think for a while and I don't know the advantage of add the old UA back...

**Comment by FrostKnight** [comment]
**This is all up to you, but I think you should focus on less UAs:**

Well, you are the one doing it anyways, so I don't think I should say much beyond this as thoughts,

ESR firefox or regular firefox, edge, chrome,  for windows

chromium firefox regular, firefox esr, for linux and unix should have same too of course...

chrome firefox reg or firefox esr  safari, for apple

chrome, firefox for android 



**firefox os, windows phone should be ditched as they are deprecated anyhow...**

console browsers im not sure about and web crawlers too,
ios is very outdated on this, so i wouldn't bother. :/

**basically, 3 definitions for the main four**,
**linux firefox esr firefox chromium
bsd same as linux
windows firefox edge chrome
mac firefox safari chrome**
unless there is some weird advantage he knows that you don't. sounds unlikely though.

**Many thanks for continuing this addon by the way! Thank you indeed!**
